run_iwyu_tests.py

```diff
@@ -206,6 +207,7 @@ def setUp(self):
       'typedef_chain_no_follow.cc': ['.'],
       'typedef_in_template.cc': ['.'],
       'typedefs_and_resugaring.cc': ['.'],
+      'typedefs_of_template_arguments.cc': ['.'],
```

Comment:
This file is not included in the patch set?

Reply:
It was added and later removed again, but I missed this line.  Good catch.